index2.html

```diff
@@ -0,0 +1,1253 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Web NFC API</title>
+  <meta charset="UTF-8">
+  <script src='https://www.w3.org/Tools/respec/respec-w3c-common'
+          class='remove'>
+  </script>
+  <script class="remove">
+    var respecConfig = {
+          specStatus:           "ED",
+          shortName:            "nfc",
+          noLegacyStyle:        true,
+          publishDate:          "",
+          previousPublishDate:  "",
+          previousMaturity:     "",
+          edDraftURI:           "http://www.w3.org/2012/nfc/web-api/",
+          // lcEnd:                "",
+          crEnd:                "",
+          editors: [
+            { name: "Kenneth Rohde Christiansen", company: "Intel",
+                    companyURL: "http://www.intel.com/" },
+            { name: "Zoltan Kis", company: "Intel",
+                    companyURL: "http://www.intel.com/" },
+          ],
+          inlineCSS:    true,
+          noIDLIn:      true,
+          // extraCSS:     ["../ReSpec.js/css/respec.css"],
+          wg:           "NFC working group",
+          wgURI:        "http://www.w3.org/2012/nfc/",
+          wgPublicList: "public-nfc",
+          wgPatentURI:  "http://www.w3.org/Consortium/Patent-Policy-20040205/",
+          otherLinks: [
+            {
+              key: "Repository",
+              data: [{
+                    value: "We are on Github.",
+                    href: "https://github.com/w3c/nfc"
+                }, {
+                    value: "File a bug.",
+                    href: "https://github.com/w3c/nfc/issues"
+                }, {
+                    value: "Commit history.",
+                    href: "https://github.com/w3c/nfc/commits/gh-pages"
+                }
+              ]
+            },
+          ]
+    };
+  </script>
+</head>
+
+<body>
+
+<!-- - - - - - - - - - - - - - - - Abstract - - - - - - - - - - - - - - - - -->
+<section id="abstract">
+  <p>
+    Near Field Communication (NFC) enables wireless communication between two
```

Comment:
this is a good generic formulation. Glossary will define more exactly (though there is variance on how close the range is, etc)